fw_gear_hcp_diff/diff_main.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import os
+import os.path as op
+
+import flywheel
+
+from fw_gear_hcp_diff.args import DiffPreprocPipeline, hcpdiff_qc_mosaic
+from utils import diff_utils, gear_preliminaries, results
+
+
+def run(context):
+    """
+    Set up and complete the Diffusion Processing stage in the HCP Pipeline.
+    """
+    # Preamble: take care of all gear-typical activities.
+    context.gear_dict = {}
+    # Initialize all hcp-gear variables.
+    gear_preliminaries.initialize_gear(context)
```

Comment:
Can't find this function,.

Reply:
It comes from hcp-base, a private gear, that creates the base Docker image and populates utils with more functions.

Reply:
I am going to pull the "shared" modules directly into this utils directory. Some of them look like they are completely superseded by GearToolkitContext calls.